.github/dependabot.yml

```diff
@@ -14,3 +14,20 @@ updates:
       security-updates:
         applies-to: security-updates
         dependency-type: production
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Why does it have to be daily ?
There is nothing so urgent about dependency upgrades that makes it necessary to do every day.
Can we make it weekly to reduce the amount of PRs this will generate ?